src/graphql/main/schema.graphql

```diff
@@ -239,9 +239,14 @@ type LnTransaction implements Transaction {
 scalar LnPaymentHash
 
 type Query {
+  globals: Globals
   me: User
 }
 
+type Globals {
+  nodesIds: [String!]!
```

Comment:
`nodeIds`?

Reply:
This is "ids of multiple nodes".

`nodeIds` might imply ids of a single node. No?

Reply:
Okay, wanted to make sure it was intentional.